sway-ir/src/block.rs

```diff
@@ -176,6 +190,49 @@ impl Block {
         }
     }
 
+    /// Replace successor `old_succ` with `new_succ`
+    pub(super) fn replace_successors(
```

Comment:
I think this should be `replace_successor()`.  The plural is a little confusing.